src/libostree/ostree-repo-commit.c

```diff
@@ -429,6 +429,20 @@ add_size_index_to_metadata (OstreeRepo        *self,
   return g_variant_ref_sink (g_variant_builder_end (builder));
 }
 
+static gboolean
+throw_min_free_space_error (OstreeRepo  *self,
+                            guint64      bytes_required,
+                            GError     **error)
+{
+  g_autofree char *formatted_free = g_format_size (bytes_required);
+  if (self->min_free_space_percent > 0)
+    return glnx_throw (error, "min-free-space-percent '%u%%' would be exceeded, %s available",
+                       self->min_free_space_percent, formatted_free);
+  else
+    return glnx_throw (error, "min-free-space-size %" G_GUINT64_FORMAT "MB would be exceeded, %s available",
```

Comment:
I tried this before but the problem is that it's not `%s available`, it's `%s requested` right?  See the 3rd hunk in this patch.  Which...I guess you are implicitly changing here.  So it seems OK, I guess we don't need to report the available space in general.

Reply:
Or, TL;DR: I think we need `s/available/requested/` here.

Reply:
hmm, yeah, Thanks for noticing. We had subtly two cases.

Reply:
So, we want to report 3rd hunk as `reserved_blocks * self->txn.blocksize` maybe ? 
(because we there is `requested` in every error reporting)